Rawl_B502/fursunnud_naile.ttl

```diff
@@ -0,0 +1,625 @@
+    irishRel:nomName "Brige".
+
+<#Bregain>
+    a foaf:Person;
+    rel:childOf <#Bráth>;
+    irishRel:nomName "Bregain".
+
+# The preceding three names, and some below, look very much like genitives
+# but I can't see why they should be, grammatically. EPT
+
+<#Bráth>
+    a foaf:Person;
+    rel:childOf <#Artt>;
+    irishRel:nomName "Bráth".
+
+<#Artt>
+    a foaf:Person;
+    rel:childOf <#Deátha>;
+    irishRel:nomName "Artt".
+
+<#Deátha>
+    a foaf:Person;
+    rel:childOf <#Elgg>;
+    irishRel:nomName "Deátha".
+
+<#Elgg>
+    a foaf:Person;
+    rel:childOf <#Arcid>;
+    irishRel:nomName "Elgg".
+
+<#Arcid>
+    a foaf:Person;
+    rel:childOf <#Alldóit>;
+    irishRel:nomName "Arcid".
+
+<#Alldóit>
+    a foaf:Person;
+    rel:childOf <#Nuadat>;
+    irishRel:nomName "Alldóit".
+
+<#Nuadat>
+    a foaf:Person;
+    rel:childOf <#Nóenail>;
+    irishRel:genName "Nuadat";
+    irishRel:nomName "Nuadu".
+
+<#Nóenail>
+    a foaf:Person;
+    rel:childOf <#ÉbirGlass>;
+    irishRel:nomName "Nóenail".
+
+<#ÉbirGlass>
+    a foaf:Person;
+    rel:childOf <#Nél>;
+    irishRel:nomName "Ébir Glass".
+
+<#Nél>
+    a foaf:Person;
+    rel:childOf <#Fóenius>;
+    irishRel:nomName "Nél".
+
+<#Fóenius>
+    a foaf:Person;
+    rel:childOf <#Glúnfhind>;
+    irishRel:nomName "Fóenius".
+
+<#Glúnfhind>
+    a foaf:Person;
+    rel:childOf <#Fethiúir>;
+    irishRel:nomName "Glúnfhind".
+
+<#Fethiúir>
+    a foaf:Person;
+    rel:childOf <#Agnoman>;
+    irishRel:nomName "Fethiúir".
+
+<#Agnoman>
+    a foaf:Person;
+    rel:childOf <#Tóe>;
+    irishRel:nomName "Agnoman".
+
+<#Tóe>
+    a foaf:Person;
+    rel:childOf <#Bainb>;
+    irishRel:nomName "Tóe".
+
+<#Bainb>
+    a foaf:Person;
+    rel:childOf <#Séim>;
+    irishRel:nomName "Bainb".
+
+<#Séim>
+    a foaf:Person;
+    rel:childOf <#Máir>;
+    irishRel:nomName "Séim".
+
+<#Máir>
+    a foaf:Person;
+    rel:childOf <#Ethecht>;
+    irishRel:nomName "Máir".
+
+<#Ethecht>
+    a foaf:Person;
+    rel:childOf <#Aurthecht>;
+    irishRel:nomName "Ethecht".
+
+<#Aurthecht>
+    a foaf:Person;
+    rel:childOf <#Aboth>;
+    irishRel:nomName "Aurthecht".
+
+<#Aboth>
+    a foaf:Person;
+    rel:childOf <#Aoy>;
+    irishRel:nomName "Aboth".
+
+<#Aoy>
+    a foaf:Person;
+    rel:childOf <#Ára>;
+    irishRel:nomName "Aoy".
+
+<#Ára>
+    a foaf:Person;
+    rel:childOf <#Iara>;
+    irishRel:nomName "Ára".
+
+<#Iara>
+    a foaf:Person;
+    rel:childOf <#Srú>;
+    irishRel:nomName "Iara".
+
+<#Srú>
+    a foaf:Person;
+    rel:childOf <#Esrú>;
+    irishRel:nomName "Srú".
+
+<#Esrú>
+    a foaf:Person;
+    rel:childOf <#Riafad>;
+    irishRel:nomName "Esrú".
+
+<#Riafad>
+    a foaf:Person;
+    rel:childOf <#Gomér>;
+    irishRel:nomName "Riafad".
+
+<#Gomér>
+    a foaf:Person;
+    rel:childOf <#Iaféth>;
+    irishRel:nomName "Gomér".
+
+<#Iaféth>
+    a foaf:Person;
+    irishRel:nomName "Iaféth";
+    rel:childOf <#Nóe>;
+    rdfs:seeAlso <https://www.wikidata.org/wiki/Q200637>.
```

Comment:
Actually, there are already owl:samAs links for biblical characters to bunad_laigen.ttl in fursunnud_naile.ttl, no?